test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14711,11 +14731,22 @@ def run(self):
                 _p.expect(out)
             self.logger.info('sending exit')
             _p.sendline("exit")
-            self.logger.info('waiting for the subprocess to finish')
-            _p.wait()
-            _p.close()
-            self.job.interactive_handle = None
-            self.logger.debug(_p.exitstatus)
+
+            if is_local:
+                self.logger.info('waiting for the subprocess to finish')
+                _p.wait()
+                _p.close()
+                self.job.interactive_handle = None
+                self.logger.debug(_p.exitstatus)
```

Comment:
why debug?

Reply:
These are not really my changes. I would guess this is just a debug message to find out if the interactive shell has terminated gracefully.